### ISSUE TITLE
Change other format than hex

### ISSUE THREAD

**Issue by guilhermetod** [filipekiss/stylelint-color-format#11]
By setting the rule as below, I thought all colors would be formatted to hsl, but it turns out only hex errors are reported and fixed. 

```
'color-format/format': {
      format: 'hsl',
},
```

Maybe we can have an option to also format the other color type (in this case, format rgb to hsl too)? I thought of something like: 

```
'color-format/format': {
      format: 'hsl',
      types: 'all' | 'hex',
},

```
